frontend/packages/console-shared/src/components/dashboard/utilization-card/UtilizationItem.tsx

```diff
@@ -72,16 +77,22 @@ export const MultilineUtilizationItem: React.FC<MultilineUtilizationItemProps> =
         domain={{ x: [startDate, endDate] }}
         loading={!error && isLoading}
         query={queries.map((q) => q.query)}
-        xAxis={false}
+        xAxis={!!isOCSRendered}
         humanize={humanizeValue}
-        padding={{ top: 13, left: 70, bottom: 0, right: 0 }}
-        height={70}
+        padding={
+          isOCSRendered
+            ? { top: 20, left: 70, bottom: 60, right: 0 }
+            : { top: 13, left: 70, bottom: 0, right: 0 }
+        }
```

Comment:
```suggestion
        padding={padding}
```
Use the original value as the default value.